tasmota/language/en_GB.h

```diff
@@ -244,8 +244,8 @@
 #define D_WEBSERVER_STOPPED "Web server stopped"
 #define D_FILE_NOT_FOUND "File Not Found"
 #define D_REDIRECTED "Redirected to captive portal"
-#define D_WIFIMANAGER_SET_ACCESSPOINT_AND_STATION "Wifimanager set AccessPoint and keep Station"
-#define D_WIFIMANAGER_SET_ACCESSPOINT "Wifimanager set AccessPoint"
+#define D_WIFIMANAGER_SET_ACCESSPOINT_AND_STATION "Wi-Fi manager set AccessPoint and keep Station"
```

Comment:
Pls do not touch en_GB unless you come up with a shorter text version. It wastes code space unneeded as the message is clear. wifi = wi-fi for apple owners.